themes/opentermsarchive/assets/css/components/article.css

```diff
@@ -0,0 +1,15 @@
+.article {
+  flex-shrink: 1;
+}
+
+.article__header {
```

Comment:
Why creating empty declarations?

Reply:
It's an inheritance from the code base, I'm going to correct it.